App.js

```diff
@@ -68,6 +69,13 @@ export default function App() {
               title="Snooze"
             />
           </Scene>
+          <Scene key="responders" type="reset" hideNavBar>
```

Comment:
would MyRespondersScreen be more appropriate under the "main" scene? As it is a page opened in that section? Please let me know what you think.

Reply:
Depends. I just made a separate "responders" scene so that the MyResponders, AddResponders, and RemoveResponders screens could all be clustered together.

If we put MyResponders in the "main" scene, it'll have the drawer component attached to it. Then I think it would make sense that all screens in "main" also be accessible through the drawer, so we'd have to add it there too.

But AddResponders and RemoveResponders will need to be put in a separate scene anyway, since you shouldn't be able to use the drawer to navigate away while in the middle of selecting responders to add/remove.

Reply:
Yes keep it this way